crates/ra_editor/src/folding_ranges.rs

```diff
@@ -132,7 +166,7 @@ fn main() {
         let folds = folding_ranges(&file);
         assert_eq!(folds.len(), 1);
         assert_eq!(folds[0].range.start(), 1.into());
-        assert_eq!(folds[0].range.end(), 48.into());
+        assert_eq!(folds[0].range.end(), 46.into());
```

Comment:
I'm not convinced by writing tests this way. Ideally, I would like to write them like this:

```rust
do_check(r#"
use std::str;
use std::vec;
use std::io as iop;
<|>use std::{
    str,
    vec,
    io as iop
};<|>"#);
```

Where each cursor indicates the beginning or the end of a fold. Or something like that.